exe/traker

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require File.expand_path(File.join(Dir.pwd, 'config/environment'))
+require 'optparse'
+require 'traker'
+
+# options = {}
+
+subtext = <<HELP
```

Comment:
To be able to cover by tests these parts, `Cli` class can be created. A class that accepts `ARGV` as arguments. So here will be only one-liner:

```ruby
Cli.new.run(args: ARGV)
```